src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/modal-views/EntityViewerSearch.tsx

```diff
@@ -17,12 +17,15 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 
+import useEntityViewerIds from 'src/content/app/entity-viewer/hooks/useEntityViewerIds';
+
 import { getEntityViewerActiveGenomeId } from 'src/content/app/entity-viewer/state/general/entityViewerGeneralSelectors';
 
 import InAppSearch from 'src/shared/components/in-app-search/InAppSearch';
 
 const EntityViewerSidebarSearch = () => {
   const activeGenomeId = useSelector(getEntityViewerActiveGenomeId);
```

Comment:
import `activeGenomeId` from `useEntityViewerIds` on the line below?

Reply:
✅